bigchaindb_driver/pool.py

```diff
@@ -63,7 +75,45 @@ def __init__(self, connections, picker_class=RoundRobinPicker):
 
         """
         self.connections = connections
+        self.node_count = len(self.connections)
+        self.max_retries = self.node_count * 4
+        self.retries = {
+            key: self.max_retries for key in range(self.node_count)
+        }
         self.picker = picker_class()
+        self.DELAY = 60
```

Comment:
60 seconds for a default delay? Any particular reason? I like what Lev proposed by adding exponential back off and starting off with a small delay.

Reply:
Is the default timeout for [tcp_fin_timeout](http://man7.org/linux/man-pages/man7/tcp.7.html)  for FIN packets on linux tcp sockets. which i considered a good constant for this case. any way i updated accordingly to @ldmberman. next push.

Reply:
@tucanae47 `tcp_fin_timeout` is about how much to wait before shutting the connection down - more like the timeout the user will pass to `forward_request` in this implementation, apparently it's different from the backoff delay.